kong/cmd/utils/prefix_handler.lua

```diff
@@ -41,6 +41,21 @@ client:request { \
 %s -e "$CMD"
 ]]
 
+local stop_script_template = [[
+#!/bin/sh
+
+CMD="\
+local conf_loader = require 'kong.conf_loader' \
+local DAOFactory = require 'kong.dao.factory' \
+local Serf = require 'kong.serf' \
+local conf = assert(conf_loader('%s')) \
+local dao = DAOFactory(conf) \
+Serf.new(conf, dao):leave() \
+"
+
+%s -e "$CMD"
+]]
```

Comment:
I meant what if the script fails, the `stop_script_template` eg. db connection failure. Then the script above ^^^ doesn't log anything.